src/styles/index.css

```diff
@@ -179,6 +183,62 @@
     --p-color-input-checked-border: var(--p-color-active);
     --p-color-input-checked-text: var(--p-color-text-default);
 
+    /* Button */
+    --p-color-button-default-bg: transparent;
+    --p-color-button-default-border: #70707B;
+    --p-color-button-default-text: var(--p-color-text-default);
+    --p-color-button-default-bg-hover: var(--p-color-button-default-bg);
+    --p-color-button-default-border-hover: var(--p-color-text-subdued);
+    --p-color-button-default-text-hover: var(--p-color-button-default-text);
+    --p-color-button-default-bg-active: #141517;
+    --p-color-button-default-border-active: var(--p-color-button-default-border-hover);
+    --p-color-button-default-text-active: var(--p-color-button-default-text-hover);
```

Comment:
Seems like most of these are the default tokens. Do we need all these button specific tokens? Possibly these are just here in case we change our mind and want these to have different values?

Reply:
Yeah for sure. Again, buttons will be a unique beast in this regard. Overall we should be able to avoid pattern level tokens like this (in this case, pattern variant level). In this case, since we're doing it, this provides us total themability if our default buttons end up having a bg in the future, etc.